packages/commonwealth/server/farcaster/router.tsx

```diff
@@ -0,0 +1,20 @@
+import express from 'express';
+import {
+  checkEligibility,
+  contestCard,
+  viewLeaderboard,
+} from './frames/contest';
+import { resultGame, startGame } from './frames/gameExample';
+
+const farcasterRouter = express.Router();
+
+farcasterRouter.get('/game', startGame);
+farcasterRouter.post('/game', startGame);
+farcasterRouter.post('/result', resultGame);
+
+farcasterRouter.get('/contestCard', contestCard);
+farcasterRouter.post('/contestCard', contestCard);
+farcasterRouter.post('/viewLeaderboard', viewLeaderboard);
+farcasterRouter.post('/checkEligibility', checkEligibility);
```

Comment:
most probably it will be used for contest work

Reply:
I think we can keep it at this level, just wondering about the model here... where are we storing farcaster related state?

Reply:
there is a way to have a shared state between multiple frames ([see here](https://framesjs.org/guides/state-management)). However, I am not sure if this is exactly what you mean.